.github/silabs-builds-mg24.json

```diff
@@ -59,7 +59,7 @@
         "platform-app": [
             {
                 "boards": ["brd4187c"],
-                "arguments": ["--output_suffix icd","--with_app 'mater_icd_core;matter'"] 
+                "arguments": ["--output_suffix icd","--with_app 'matter_icd_core;matter'"] 
```

Comment:
```suggestion
                "arguments": ["--output_suffix icd","'--with_app matter_icd_core;matter'"] 
```

Reply:
When micheal added the `--with_*` flags, the escaping wasn't done correctly to match the json structure. It was fixed after the PR was merged. 

The linked example is the incorrect espacing before the fix was merged.